checkov/main.py

```diff
@@ -393,6 +396,11 @@ def add_parser_args(parser):
         ),
         default=None,
     )
+    parser.add('--min-cve-severity', help='Set minimum severity that will cause returning non-zero exit code',
+               choices=SEVERITY_RANKING.keys(), default='none')
+    parser.add('--skip-cve-package',
+               help='filter scan to run on all packages but a specific package identifier (denylist), You can '
+                    'specify multiple packages separated by comma delimiter', action='append', default=None)
```

Comment:
as far as I remember `action='append'` means that you can use the flag several times and the values will be appended to a list. `--skip-cve-package flask --skip-cve-package django`